docs/setuptools.txt

```diff
@@ -2293,6 +2293,44 @@ Metadata and options are set in the config sections of the same name.
 
 * Unknown keys are ignored.
 
+setup.cfg-only projects
+=======================
+
+.. note:: New in 40.9.0 (03 Apr 2019).
+
+Latest versions of ``setuptools`` emulate a dummy ``setup.py`` module
+containing only a ``setuptools.setup()`` call.
+
+This means that you can have a Python distribution project with no
+``setup.py`` but having only ``setup.cfg`` present, given you **can rely
+on** having ``setuptools >= 40.9.0`` being present prior the build.
+
+This can happen if:
+
+* the target Python version is up-to-date and bundles sufficient
+  ``setuptools`` version
+
+* the target env has ``pip >= 19`` (or any other compatible front-end)
+  which has support for PEP 517 and your project uses ``pyproject.toml``
+  to specify ``setuptools >= 40.9.0`` as a build dependency.
+  For example:
+
+  .. code-block:: toml
+
+      [build-system]
+      requires = [
+        "setuptools >= 40.9.0",
+        "wheel",
+      ]
+      build-backend = "setuptools.build_meta"
+
+  .. warning::
+
+      Beware that due to a `Pip bug #6264`_, system site-packages may
```

Comment:
I'm not sure how much I want to bake information about `pip` bugs into the documentation of `setuptools`.

Reply:
I see. But OTOH if a user finds the info about the integration in this place they'll be likely returning here to see what they've done wrong in case of issues. Which makes this section a perfect place to mention the bug.